src/ddmd/traits.d

```diff
@@ -370,6 +373,73 @@ extern (C++) Expression pointerBitmap(TraitsExp e)
     return ale;
 }
 
+extern (C++) Expression externDmangle(TraitsExp e, Scope* sc)
```

Comment:
Added comment and a test case. I was rather surprised rereading the code I wrote a year ago ;-) Being tailored for functions and the use in externDfunc, I guess its usefullness is limited and the name is misleading.
Maybe it's better to actually return the mangled string, but that will still require the current trickery in core.internal.traits.externDFunc.